openBCIBoard.js

```diff
@@ -472,6 +472,8 @@ function OpenBCIFactory() {
                 reject('Simulator Already Running');
             } else {
                 // start simulating
+                this.connected = true;
+                this.streaming = true;
                 this.isSimulating = true;
                 // generateSample is a func that takes the previous sample number
                 var generateSample = openBCISample.randomSample(this.numberOfChannels(),this.sampleRate());
```

Comment:
where is this.sampleRate coming from?  I can't seem to find it.  Also, can we make the sample rate a parameter that defaults to 250?  Austin was saying that 250 samples per second might overwhelm the browser, so having an option to simulate a slow rate would be useful as we start to implement the real-time plotting